arm-sql/2014-04-01/swagger/sql.core.json

```diff
@@ -294,7 +294,7 @@
           "Databases"
         ],
         "operationId": "Databases_ListByServer",
-        "description": "Returns information about an Azure SQL database.",
+        "description": "Returns information about a database.",
```

Comment:
I know this isn't your error, but could you put "Returns a list of databases by server."?